src/@next/Radio/RadioStyle.ts

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+import { Colors } from '..';
+import { ScreenSize } from '../../Utils/StyleConfig';
+
+export const RadioWrapper = styled.div`
+  display: flex;
+  align-items: flex-start;
+  gap: 8px;
+`;
+
+export const RadioInput = styled.input`
+  cursor: pointer;
+  appearance: none;
+  margin: 0;
+  width: 18px;
+  height: 18px;
+  border: 2px solid ${Colors.Neutral.B68};
+  border-radius: 50%;
+  transform: translateY(1.5px);
+
+  ::after {
+    content: '';
+    display: block;
+    border-radius: 50%;
+    width: 8px;
+    height: 8px;
+    margin: 3px;
+  }
+
+  @media (min-width: ${ScreenSize.tablet}px) {
+    width: 22px;
+    height: 22px;
+    transform: none;
+    ::after {
+      width: 10px;
+      height: 10px;
+      margin: 4px;
+    }
+  }
+
+  :hover {
+    border-color: ${Colors.Neutral.B68};
+  }
+
+  :focus,
+  :focus-visible,
+  :focus-within {
+    box-shadow: 0px 0px 0px 1px ${Colors.Neutral.B100},
+      0px 0px 0px 3px ${Colors.Blue.S54};
+    outline: none;
+  }
+
+  :disabled {
+    cursor: not-allowed;
+    background-color: ${Colors.Neutral.B95};
+    border-color: ${Colors.Neutral.B85};
+  }
+
+  :checked {
+    border-color: ${Colors.Blue.S99};
+
+    ::after {
+      background-color: ${Colors.Blue.S99};
+    }
+
+    :disabled {
+      background-color: ${Colors.Neutral.B95};
+      border-color: ${Colors.Neutral.B85};
+
+      ::after {
+        background-color: ${Colors.Neutral.B85};
+      }
+    }
+  }
+`;
+
+export const LabelWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`;
+
+type LabelProps = {
+  disabled?: boolean;
+};
+
+export const Label = styled.label<LabelProps>`
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+
+  ${({ disabled }) => {
+    console.log('disabled', disabled);
+    return (
+      disabled &&
+      `
+        span {
+          color: ${Colors.Neutral.B85} !important;
```

Comment:
can we avoid using !important but use this instead: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
Fixed